subsys/bluetooth/audio/audio.c

```diff
@@ -67,6 +67,15 @@ int bt_audio_data_parse(const uint8_t ltv[], size_t size,
 	return 0;
 }
 
+uint8_t bt_audio_get_chan_count(enum bt_audio_location chan_allocation)
+{
+	if (chan_allocation == BT_AUDIO_LOCATION_MONO_AUDIO) {
+		return 1;
+	}
+
+	return POPCOUNT(chan_allocation);
+}
```

Comment:
Suggest to either define POPCOUNT at the right place, or do something like
```suggestion
{
	if (chan_allocation == BT_AUDIO_LOCATION_MONO_AUDIO) {
		return 1;
	}
#if defined(POPCOUNT)
	return POPCOUNT(chan_allocation);
#else
	/* loop */
#endif
}
```

Alternatively we can also just omit POPCOUNT complete and just rely on a simple loop.